src/Application.vala

```diff
@@ -35,12 +33,12 @@ public class Screenshot.ScreenshotApp : Gtk.Application {
     public ScreenshotApp () {
         Object (
             application_id: "io.elementary.screenshot-tool",
-            flags: ApplicationFlags.HANDLES_COMMAND_LINE
+            flags: ApplicationFlags.FLAGS_NONE
         );
     }
 
     construct {
-        options = new OptionEntry[7];
+        var options = new OptionEntry[7];
```

Comment:
Can you also change that to be a `const OptionEntry` like in https://github.com/elementary/installer/blob/7a3ee9a718966db0bcec2d420e1cbf567afcbb3a/src/Application.vala#L22
That's more efficient that way ;)

Reply:
I'm getting errors from the compiler that the options are not constant because of the translatable strings. Is it acceptable to lose those translations?